ckanext/switzerland/templates/user/snippets/login_form.html

```diff
@@ -0,0 +1,32 @@
+{#
+Renders the login form.
+
+action        - The url that the form should be submitted to.
+error_summary - A tuple/list of form errors.
+
+Example:
+
+  {% snippet "user/snippets/login_form.html", action=g.login_handler, error_summary=error_summary %}
+
+#}
+{% import 'macros/form.html' as form %}
+
+{% set username_error = true if error_summary %}
+{% set password_error = true if error_summary %}
+
+<form action="{{ action }}" method="post" class="dataset-form">
+  {{ h.csrf_input() }}
+  {{ form.errors(errors=error_summary) }}
+
+  {{ form.input('login', label=_("Username or Email"), id='field-login', value="", error=username_error, classes=["control-medium"]) }}
+
+  {{ form.input('password', label=_("Password"), id='field-password', type="password", value="", error=password_error, classes=["control-medium"]) }}
+
+  {{ form.checkbox('remember', label=_("Remember me"), id='field-remember', checked=true, value="63072000") }}
```

Comment:
I'm not sure - this part of the template was copied from CKAN core. (Actually, the only thing I changed here was to add `class="dataset-form"` to the `<form>`.) That value has been included in this template since 2012 so I guess it's necessary, although that number is not found anywhere else in the code. :sweat_smile: